src/routes/__layout.svelte

```diff
@@ -121,6 +122,14 @@
             isSelected="{$page.url.pathname === '/admin'}"
           />
         {/if}
+        {#if $session?.user?.isStaff}
```

Comment:
this should get the same treatment as the route to allow Guild Owner, Admins, and Staff. We could also use the `getUserAccess` function as a template for a separate `getUserSessionAccess` (for example)